gpt4all-bindings/cli/app.py

```diff
@@ -103,7 +114,8 @@ def repl(
 
 @app.command()
 def version():
-    print("gpt4all-cli v0.3.4")
+    """The CLI version commmand."""
```

Comment:
spellcheck

Reply:
Good catch! Somehow my plugin didn't show me that one. Might've accepted it by accident.